Sources/Auth/Types/AuthPayloadParams.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+struct AuthPayloadParams {
+    let type: String
+    let chainId: String
+    let domain: String
+    let aud: String
+    let version: String
+    let nonce: String
+    let iat: String
+    let nbf: String?
+    let exp: String?
```

Comment:
I still defend the approach of using readable names and declaring the abbreviated coding keys 😬

Reply:
That’s according to cacao specs :-)